### PR TITLE
Reorganize, rewrite, & add information about install fields in harvester-configuration.md

### PR DESCRIPTION
**Pull request by jillian-maroket**
Related issue: [#5146](https://github.com/harvester/harvester/issues/5146)

Scope: Install fields (v1.2, v1.3, v1.4)

Changes:
- Listed fields in alphabetical order
- Created standard information types (Definition, Versions, Fields, Default value, Supported values, Example)
- Rewrote some content to match information types and correct language errors
- Changed markup (removed unnecessary headings and notes, etc.)
- Wrote definitions for a few undocumented fields
  - `automatic`, `rawdiskimagepath`, `wipedisks` - SME: @Vicente-Cheng 
  - `webhooks`, `withnetimages` - SME: @bk201

### CODE REVIEW

File: docs/install/harvester-configuration.md [hunk 1]
```diff
@@ -439,335 +437,372 @@ os:
...
-In edge scenarios where users may deploy single-node Harvester clusters, can set this parameter to a value smaller than 12. In most scenarios, it is recommended to keep the default value for system high availability.
+**Fields**:
 
-Before setting the value, please refer to [longhorn-guaranteed-replica-manager-cpu](https://longhorn.io/docs/1.4.1/references/settings/#guaranteed-replica-manager-cpu) for more details.
+- `event`: Event type that triggers an HTTP action on the webhook. 
+  - `STARTED`: The installation has started.
+  - `SUCCEEDED`: The installation was completed without errors.
+  - `FAILED`: The installation was unsuccessful.
+- `method`: HTTP method 
+- `url`*: URL to which HTTP requests are sent
```
Comment (bk201) [inline docs/install/harvester-configuration.md:761]:
```suggestion
- `url`: URL to which HTTP requests are sent
```

**Comment by jillian-maroket** [comment]
Hello, @m-ildefons and @tserong. Please check the changes and let me know how we can further improve the descriptions. 

Preview: https://6617d4c18e4b1f8f09c7fa2e--harvester-preview.netlify.app/v1.4/install/harvester-configuration#installautomatic

Notes:
- New: `webhooks`, `withnetimages`
- Still undocumented: `automatic`, `rawdiskimagepath`, `wipedisks`
- Will not be documented (per Kiefer's suggestion): `configurl`, `forcegpt`

**Comment by Vicente-Cheng** [comment]
LGTM with these three parameters: `automatic`, `rawdiskimagepath`, `wipedisks`

**Review by bk201 — APPROVED** [review]
LGTM, thank you!

**Pull request merged**